storage/innobase/handler/handler0alter.cc

```diff
@@ -654,9 +654,13 @@ static MY_ATTRIBUTE((warn_unused_result)) bool innobase_need_rebuild(
   Alter_inplace_info::HA_ALTER_FLAGS alter_inplace_flags =
       ha_alter_info->handler_flags & ~(INNOBASE_INPLACE_IGNORE);
 
-  if (Encryption::none_explicitly_specified(
-          ha_alter_info->create_info->explicit_encryption,
-          ha_alter_info->create_info->encrypt_type.str) ||
+  if ((Encryption::none_explicitly_specified(
```

Comment:
Please also consider using two helper `const bool` variables to make this if more readable / understandable.

Reply:
done